backend/src/enums/user.role.ts

```diff
@@ -1,4 +1,10 @@
 export enum UserRole {
-  ADMIN = 'admin',
-  USER = 'user',
+  // can change roles, delete users, etc.
+  ADMIN = 100,
+
+  // can delete users, create/delete/modify boards, etc.
+  MEMBER = 10,
```

Comment:
I don't like this name at all, maybe `MODERATOR`?

Reply:
I'm also not sure if we need this role

Reply:
i'll delete this role